apps/src/music/music.module.scss

```diff
@@ -0,0 +1,84 @@
+@import "./theme/tokens-dark";
+
+// Constants
+$timeline-area-height: 200px;
+$default-border-radius: 4px;
+
+.container {
+  position: relative;
+  background-color: black;
+  color: white;
+  width: 100%;
+  height: 100%;
+  padding: 0;
+  box-sizing: border-box;
+  border-radius: $default-border-radius;
+  overflow: hidden;
+  user-select: none;
+  display: flex;
+  flex-direction: column;
+}
+
+.instructionsArea {
+  display: flex;
+  color: white;
+  background-color: black;
+  border-radius: $default-border-radius;
+  padding: 0;
+  box-sizing: border-box;
+
+  &.instructionsTop {
+    flex-direction: row;
+    height: 90px;
+    margin-bottom: 5px;
+    width: 100%;
+  }
+
+  &.instructionsSide {
+    flex-direction: column;
+    height: 100%;
+    width: 20%;
+
+    &.instructionsLeft {
+      margin-right: 10px;
+    }
+
+    &.instructionsRight {
+      margin-left: 10px;
+    }
```

Comment:
Oh, could we reduce these to `5px` now that all the other gaps are also smaller now?